newsService/src/main/java/com/kovan/app/service/NewsService.java

```diff
@@ -78,10 +76,10 @@ public List<NewsDto> getTopHeadlines() {
 
         Map<Boolean, List<NewsDto.Article>> partitionedArticles = articles.stream()
                 .takeWhile(article -> !parseDate(article.getPublishedAt()).isBefore(yesterday))
-                .collect(Collectors.partitioningBy(article -> parseDate(article.getPublishedAt()).equals(yesterday)));
+                .collect(partitioningBy(article -> parseDate(article.getPublishedAt()).equals(yesterday)));
 
-        List<NewsDto.Article> newsArticles = createList(partitionedArticles.get(true));
-        List<NewsDto.Article> futureArticles = createList(partitionedArticles.get(false));
+        List<NewsDto.Article> newsArticles = partitionedArticles.get(true);
+        List<NewsDto.Article> futureArticles = partitionedArticles.get(false);
 
         if (newsArticles.isEmpty() && futureArticles.isEmpty() && !isYesterdayInDb) {
```

Comment:
Yes. It will return an empty list if there is no value. 
![Screenshot (151)](https://github.com/user-attachments/assets/ca9af79f-6b95-4299-b214-defa163a939b)